### ISSUE TITLE
Data transfer failed from bigquery to snowflake

### ISSUE THREAD

**Issue by eyepax-ChandanaM** [slingdata-io/sling-cli#174]
Hi there,
I was able to configure and run data transfer command from google bigquery to snowflake successfully from sling cli.
but when I set the **gc_bucket** option in biquery connection config the data transfer job failed. [error.log](https://github.com/slingdata-io/sling-cli/files/14309162/error.log). 

```
100039 (22003): Numeric value '170712627001774500' is out of range
  File '"PUBLIC"."ADFORM_ENV_TMP"/2024-02-16T150145.423/part.01.0001.csv.zst', line 3, character 39
  Row 2, column "ADFORM_ENV_TMP"["TRANSACTION_ID":3]
  If you would like to continue loading when an error is encountered, use other values such as 'SKIP_FILE' or 'CONTINUE' for the ON_ERROR option. For more information on loading options, please run 'info loading_data' in a SQL client.

```

Please note that data transfer is completed successfully once after removing the gc_bucket config from the bigquery connection config.

Note:
I'm using` --mode full-refresh` here.

**Comment by flarco** [comment]
@eyepax-ChandanaM thanks.

Could share two more outputs?
1. Output of run with `gc_bucket`
2. Output of run without `gc_bucket`

Also, please use the `-d` flag to show more debug information, like this:
`sling run --src-conn BIGQUERY --src-stream "select * from ad_measurement_data.adform_impressions" --tgt-conn SNOWFLAKE --tgt-object "public.adform_env" --mode full-refresh -d`

**Comment by eyepax-ChandanaM** [comment]
Hi @flarco 

Please find the logs here.
[error_log_with_bucket_config.txt](https://github.com/slingdata-io/sling-cli/files/14310575/error_log_with_bucket_config.txt)
[success_log_without_bucket_config.txt](https://github.com/slingdata-io/sling-cli/files/14310576/success_log_without_bucket_config.txt)

**Comment by flarco** [comment]
Thanks, I was able to reproduce.
Fixed in #173  (v1.1.6)
Will release soon.

**Comment by AdMatch-Chandana** [comment]
Thanks
